src/vsr.zig

```diff
@@ -590,6 +763,76 @@ pub fn parse_addresses(allocator: *std.mem.Allocator, raw: []const u8) ![]std.ne
     return addresses[0..index];
 }
 
+test "parse_addresses" {
+    const test_successes = &[_]struct{
+        raw: []const u8,
+        addresses: [3]std.net.Address,
+    }{
+        .{
+            // Test the minimum/maximum port.
+            .raw = "1.2.3.4:0,2.3.4.5:789,3.4.5.6:65535",
```

Comment:
Great testing of the 0 and max 16-bit boundaries here.

Let's do the same for the IPv4 components as well, have one address of "0.0.0.0", another of "255.255.255.255", and one where the components are all distinct like we have already "1.2.3.4"?